DiaHelperApplication/DiaHelperApplication/Controllers/AddNutritionViewController.swift

```diff
@@ -7,21 +7,317 @@
 
 import UIKit
 
-class AddNutritionViewController: UIViewController {
+protocol AddNutritionViewControllerDelegate: AnyObject {
+    func didAddMeal(_ meal: Meal)
+}
+
+class AddNutritionViewController: UIViewController, UITextFieldDelegate {
+    weak var delegate: AddNutritionViewControllerDelegate?
+    private let tableView = UITableView()
```

Comment:
Similar to the Glucometer VC, a StackView is usually an easier approach for static forms